apps/web/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,23 @@
+import { Flex, Link } from "@chakra-ui/react";
+
+import colors from "../../styles/colors";
+
+export const Footer = () => (
+  <Flex color={colors.grey[600]} lineHeight="1">
+    © 2024 Umami
+    <Link
+      margin="0 8px"
+      padding="0 8px"
+      borderColor={`${colors.grey[300]} !important`}
+      borderRight="1px solid"
+      borderLeft="1px solid"
+      href="https://umamiwallet.com/tos.html"
+      target="_blank"
```

Comment:
this thing is [quite dangerous](https://docs.datadoghq.com/code_analysis/static_analysis_rules/jsx-react/jsx-no-target-blank/). let's at least use the `isExternal` prop